src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,68 @@
+package duke.ui;
+
+import java.util.Scanner;
+
+public class Ui {
+    private final String logo = " ____        _        \n" + "|  _ \\ _   _| | _____ \n" + "| | | | | | | |/ / _ \\\n"
+        + "| |_| | |_| |   <  __/\n" + "|____/ \\__._|_|\\_\\___|\n";
+    private final String logoDivider = "++++++++++++++++++++++\n";
```

Comment:
Please abide by the naming convention for `final` variables (e.g. UI_LOGO, UI_LOGO_DIVIDER).